src/main/java/io/antmedia/ipcamera/IOnvifCamera.java

```diff
@@ -16,7 +16,7 @@
 	 * @return true if connected
 	 * 		   false if not connected
 	 */
-	boolean connect(String address, String username, String password);
```

Comment:
Please write some comments about return type. It returns what and meaning of the it